mpt_trie/src/nibbles.rs

```diff
@@ -26,6 +26,11 @@ pub type Nibble = u8;
 /// Used for the internal representation of a sequence of nibbles.
 pub type NibblesIntern = U512;
 
+const NIBBLES_APPEND_ASSERT_ERR_MSG: &str =
```

Comment:
How do you feel about `MULTIPLE_NIBBLES... `rather than `NIBBLES...` and `SINGLE_NIBBLE...` rather than `NIBBLE...`? They're not exactly easy to distinguish right now

Reply:
Yeah agreed.